shared/asset.js

```diff
@@ -184,12 +184,12 @@ export function getType(asset, publications, wallet) {
   return TYPES.taken
 }
 
-export function isValidName(name) {
-  return name <= 50
+export function isValidName(name = '') {
+  return name.length > 0 && name.length <= 50
```

Comment:
This could be `name.length && name.length <= 50`

Reply:
TypeScript doesn't think the same :P although javascript allows stuff like that let's stick to consistent return values (in this case, always return a boolean)